Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -487,7 +487,12 @@ extension BrowserViewController: BrowserToolbarDelegate {
     func browserToolbarDidPressShare(browserToolbar: BrowserToolbarProtocol, button: UIButton) {
         if let selected = tabManager.selectedTab {
             if let url = selected.displayURL {
-                var activityViewController = UIActivityViewController(activityItems: [selected.title ?? url.absoluteString!, url], applicationActivities: nil)
+                let printInfo = UIPrintInfo(dictionary: nil)
+                printInfo.jobName = url.absoluteString
+                printInfo.outputType = .General
```

Comment:
Do we need the printInfo here? I removed it locally and things seem fine (but I don't have an AirPrint compatible printer).